ExerciseOne/exercise-one.html

```diff
@@ -1,160 +1,102 @@
 <!DOCTYPE html>
+<html>
 <head>
-    <script type="text/javascript">
-        console.log("javascript code goes here")
-    </script>
+    <title>Nike Store Tab Title</title>
+    <style>
+        .logo {
+            display: block;
+            margin: 0 auto;
+            width: 100px;
+            }
```

Comment:
Why is this end bracket indented like this?  Please always check other similar-elements in CSS to see closing brackets } align